src/provider/location.ts

```diff
@@ -11,5 +11,5 @@ export interface Location {
   city?: string
   asn?: number
   isp?: string
-  node_type?: string
+  nodeType?: string
```

Comment:
Shouldn't it be `node_type`?

Reply:
It automatically converts to camelCase for JS users

Reply:
From `tequilapi-client-factory-ts`:

```js
  public axiosInstance(): AxiosInstance {
    const ax = axios.create({
      baseURL: this._baseUrl,
      headers: {
        'Cache-Control': 'no-cache, no-store',
      },
    })
    const convertOptions = {
      deep: true,
    }
    ax.interceptors.request.use((config) => {
      if (config.params) {
        config.params = snakecaseKeys(config.params, convertOptions)
      }
      if (config.data) {
        config.data = snakecaseKeys(config.data, convertOptions)
      }
      return config
    })
    ax.interceptors.response.use((config) => {
      if (config.data) {
        config.data = camelcaseKeys(config.data, convertOptions)
      }
      return config
    })
    return ax
```